distributed/worker.py

```diff
@@ -4576,3 +4591,74 @@ def warn(*args, **kwargs):
         worker.log_event("warn", {"args": args, "kwargs": kwargs})
 
     warnings.warn(*args, **kwargs)
+
+
+def benchmark_disk(
+    sizes=["1 kiB", "100 kiB", "1 MiB", "10 MiB", "100 MiB"],
+    rootdir=None,
+    duration=1.0,
+) -> dict:
+    duration = parse_timedelta(duration)
+
+    out = {}
+    for size_str in sizes:
+        with tmpdir(dir=rootdir) as dir:
+            dir = pathlib.Path(dir)
+            names = list(map(str, range(100)))
+            size = parse_bytes(size_str)
+
+            data = os.urandom(size)
```

Comment:
1. ``secrets.token_bytes(size)`` is preferrable
2. ``numpy.radom.randint(255, size, dtype="u8").tobytes()`` is faster (but numpy may not be installed)
3. ``random.randbytes(size)`` is even faster, but requires Python 3.9
```python
%timeit os.urandom(100*2**20)
2.13 s ± 53.2 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
%timeit secrets.token_bytes(100*2**20)
2.11 s ± 17.2 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
%timeit numpy.random.randint(255, size=100*2**20, dtype="u8").tobytes()
621 ms ± 5.41 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
%timeit random.randbytes(100*2**20)
322 ms ± 3.98 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
```

Reply:
It looks like `os.urandom` and `secrets.token_bytes` are the same speed.  I suspect that token_bytes is more secure, but in our case we don't care.  Is there a reason to prefer token_bytes?

Reply:
The difference between os.urandom and secrets.token_bytes is mostly phylosophical - the latter is a higher level abstraction than the former.

We should definitely use random.randbytes -> numpy -> secrets.token_bytes/urandom though

Add to compatibility.py:
```python
if sys.version_info >= (3, 9):
    from random import randbytes
else:
    try:
        import numpy
        def randbytes(size):
            return numpy.random.randint(255, size=size, dtype="u8").tobytes()
     except ImportError:
        def randbytes(size):
            return secrets.token_bytes(size)
```

Reply:
(also please search/replace for os.urandom, np.random.randint, and numpy.random.randint)

Reply:
Added randbyets